lib/IDE/CodeCompletion.cpp

```diff
@@ -5952,7 +5953,8 @@ static void addAnyTypeKeyword(CodeCompletionResultSink &Sink, Type T) {
 }
 
 void CodeCompletionCallbacksImpl::addKeywords(CodeCompletionResultSink &Sink,
-                                              bool MaybeFuncBody) {
+                                              bool MaybeFuncBody,
+                                              bool ForCallArgExpr) {
```

Comment:
I don't think it's worth to call this function from the main `switch`. It can directly call
```cpp
      addSuperKeyword(CompletionContext.getResultSink());
      addExprKeywords(CompletionContext.getResultSink());
```
from the each `case` branch. What do you think?

Reply:
Yeah, that's fair enough